src/teleop_twist_joy.cpp

```diff
@@ -216,3 +219,5 @@ void TeleopTwistJoy::Impl::joyCallback(const sensor_msgs::msg::Joy::SharedPtr jo
 }
 
 }  // namespace teleop_twist_joy
+
+RCLCPP_COMPONENTS_REGISTER_NODE(teleop_twist_joy::TeleopTwistJoy)
```

Comment:
We've been putting the components include right above this line, but I don't think that it matters much.

Reply:
Yeah, it makes copy-n-paste easier.  But I don't really love putting a random `#include` at the end of source code, so I think I'll leave it as-is for now.